sky/serve/constants.py

```diff
@@ -9,3 +9,8 @@
 
 CONTROL_PLANE_PORT = 31001
 CONTROL_PLANE_SYNC_INTERVAL = 20
+
+CONTROLLER_RESOURCES = {'disk_size': 100, 'cpus': '4+'}
+
+DEFAULT_INITIAL_DELAY_SECONDS = 1200
```

Comment:
Add a comment for `DEFAULT_INITIAL_DELAY_SECONDS` to briefly explain what it means?